setup.py

```diff
@@ -0,0 +1,141 @@
+"""
+setuptools-based build of ROOT.
+
+This script uses setuptools API to steer a custom CMake build of ROOT. All build
+configuration options are specified in the class responsible for building. The
+installation step produces the following output target install directory:
+
+```
+site-packages/
+-- cppyy/
+-- cppyy_backend/
+-- libcppyy.so
+-- libcppyy_backend.so
+-- ROOT/
+-- -- libROOTPythonizations.so
```

Comment:
```suggestion
site-packages/
-- cppyy/
-- libcppyy.so
-- ROOT/
-- -- libROOTPythonizations.so
-- -- __init__.py (and other Python sources)
-- -- _pythonization/
```
Just a small nitpick: the `ROOT/` directory contains more than `libROOTPythonizations.so`.

Also, if we would decide to go ahead with #19470, the backend is gone.